Dockerfile

```diff
@@ -11,8 +12,13 @@ COPY ["run-periodically.js", "run-periodically.js"]
 
 # Source files directories
 COPY ["server", "server"]
+COPY ["config", "config"]
 
-RUN npm install --production --no-optional
+RUN apk --no-cache --virtual build-dependencies add \
+    python \
+    make \
+    && npm ci --production \
+    && apk del build-dependencies
```

Comment:
Is this necessary?

Reply:
That was my impression. I can try to remove all the black magic once more and see if it flies.

Reply:
Seems to build even after the removal, so I'll remove it :D